### DISCUSSION TITLE
How to increase dpi or figure size when plotting?

### DISCUSSION THREAD

**Discussion by soldivelot** [colour-science/colour#967] [General]
### Question

it looks a little crowded

![colourspaces](https://user-images.githubusercontent.com/84898012/160537251-34e06867-f035-4c6c-a099-fc79faf3bdc2.svg)

I would like to know how to change it to something like this

![doc](https://colour.readthedocs.io/en/develop/_images/Plotting_Plot_RGB_Colourspaces_In_Chromaticity_Diagram_CIE1976UCS.png)

**Comment by thomasmansencal** [comment]
The handy [How-To Guide](https://colab.research.google.com/drive/1NRcdXSCshivkwoU2nieCvC3y14fx1X4X#offline=true&sandboxMode=true) hints at the very top on how to do that. Basically, simply use a call to `matplotlib.pyplot` as follows:

```python
import matplotlib.pyplot as plt

plt.style.use({'figure.dpi': 100})
plt.style.use({'figure.figsize': (12.80, 7.20)})
```

Alternatively, a simple call to `colour.plotting.colour_style` will set defaults that we think are appropriate for a good number of use cases.

**Reply by soldivelot** [reply]
solved, thanks :smile:
